src/plugins/timelist/Timelist.vue

```diff
@@ -68,7 +68,7 @@ const headerItems = [
         defaultDirection: false,
         property: 'duration',
         name: 'Time To/From',
-        format: function (value) {
+        format: function (value, object) {
```

Comment:
Do we need the object here?

Reply:
Nope, definitely not a requirement. Currently there is the same unused `object` param in the `Start Time`, `End Time` header items. I initially added object to `Time To/From` for troubleshooting purposes, but left it to be consistent with the other two items. @shefalijoshi - would you recommend removing all 3 unused `object` params?